css/scss/layout/_main.scss

```diff
@@ -0,0 +1,63 @@
+main {
```

Comment:
Looks cools, but i think if would be great if you add min-height for this container or smth that will stick footer to the bottom.
![](https://i.imgur.com/7sERtQ0.png)

Reply:
Looks like everything is done